ddsrouter_core/src/cpp/communication/Track.cpp

```diff
@@ -136,15 +136,15 @@ void Track::no_more_data_available_() noexcept
 
     // It may occur that within the process of set data_available_status, the actual status had changed
     // Thus, it must take care that it is only set to NO_DATA when it comes from transmitting data
-    if (data_available_status_ == DataAvailableStatus::TRANSMITTING_DATA)
+    if (data_available_status_ == DataAvailableStatus::transmitting_data)
     {
         logDebug(DDSROUTER_TRACK, "Track " << *this << " has no more data to send.");
-        data_available_status_.store(DataAvailableStatus::NO_MORE_DATA);
+        data_available_status_.store(DataAvailableStatus::no_more_data);
     }
-    // If it is NEW_DATA_ARRIVED is that the Listener has notified new data AFTER Track has received a NO_DATA
+    // If it is DataAvailableStatus::new_data_arrived is that the Listener has notified new data AFTER Track has received a NO_DATA
```

Comment:
**NIT**
**In general** 
it looks like using replace has also affected the comments. If it is not much work, please undo these changes.